static/js/main.js

```diff
@@ -76,15 +71,36 @@ const createInputs = () => {
     return inputs;
 };
 
+const createDropDown = () => {
+  let select_X = document.getElementById("x-values-input")
+  let select_Y = document.getElementById("y-values-input")
+  let inputs = ['Voltage', 'Length', 'r0', 'ra', 'x', 'Force', 'AWG']
+  inputs.forEach( element => {
+    let option = document.createElement('option')
+    option.text = `${element}`
+    option.value = `${element}`
+    option.id = `option-x-${element}`
+    select_X.add(option)
+    if(element !== 'AWG'){
+      let option1 = document.createElement('option')
+      option1.text = `${element}`
+      option1.value = `${element}`
+      option1.id = `option-y-${element}`
+      select_Y.add(option1)
+    }
+  })
+}
+
 const populateDefaults = () => {
   document.getElementById('input-text-voltage').value = '5';
   document.getElementById('input-text-length').value = '27';
-  document.getElementById('input-text-r_not').value = '2.3';
-  document.getElementById('input-text-r_a').value = '4.5';
+  document.getElementById('input-text-r0').value = '2.3';
+  document.getElementById('input-text-ra').value = '4.5';
   document.getElementById('input-text-x').value = '0';
   document.getElementById('input-text-awg').value = '30';
   document.getElementById('input-text-force').value = '';
-  document.getElementById('input-radio-voltage').checked = true;
+  document.getElementById('x-values-input').value = 'Voltage';
+  document.getElementById('y-values-input').value = 'Force';
 }
```

Comment:
Can we add a test for this? Not only the `populateDefaults()` function but also something to change those parameters for the `x-values-input` and the `y-values-input`

Reply:
I think this is worthwhile, yes. I can work with Nate and add those tests

Reply:
For changing the x and y values, we can change them and assert that they were changed, but that is the extent that we can do right now because of how that chart is rendered. Right now the way the code is, there is no way to assert the correct chart was actually rendered. Do you still want a test for just changing the values and asserting they were changed? The populate defaults I completely agree with

Reply:
Right, I'm aware about the chartjs problem, that's fine. But I think we should also assert the changes for when X and Y are changed:
Verify that when Length is selected for the X dropdown that it is not in the Y dropdown and vice-versa. Those kinds of things

Reply:
Okay yeah I understand now. Completely agree